api/test/logs/provider_test.cc

```diff
@@ -57,15 +62,16 @@ TEST(Provider, MultipleLoggerProviders)
 TEST(Provider, GetLogger)
 {
   auto tf = shared_ptr<LoggerProvider>(new TestProvider());
-  // tests GetLogger(name, options)
-  auto logger = tf->GetLogger("logger1");
+  // tests GetLogger(name, version, schema)
+  const std::string schema_url{"https://opentelemetry.io/schemas/1.2.0"};
+  auto logger = tf->GetLogger("logger1", "", "library_name", "", schema_url);
   EXPECT_EQ(nullptr, logger);
 
   // tests GetLogger(name, arguments)
 
-  std::array<string_view, 1> sv{"string"};
-  span<string_view> args{sv};
-  auto logger2 = tf->GetLogger("logger2", args);
+  std::array<nostd::string_view, 1> sv{"string"};
+  nostd::span<nostd::string_view> args{sv};
+  auto logger2 = tf->GetLogger("logger2", args, "lib_name", "", schema_url);
```

Comment:
nit: use another fake library name like `opentelelemtry_library` and make it consistent in all the tests?

Reply:
done